x-pack/plugin/apm-data/src/main/resources/component-templates/apm@mappings.yaml

```diff
@@ -12,6 +13,8 @@ template:
           type: scaled_float
           scaling_factor: 1000000
     properties:
+      system.process.cpu.start_time:
+        type: date
```

Comment:
Could this be scoped to [metrics-apm@mappings](https://github.com/elastic/elasticsearch/blob/5cc6e2fa34387bdf05126b7cf89f08f3906b2b09/x-pack/plugin/apm-data/src/main/resources/component-templates/metrics-apm%40mappings.yaml)? I'm not aware of this field being set outside metrics.